src/webpack.js

```diff
@@ -160,20 +162,65 @@ const watchBundleChanges = (devMode, signingKey, callback, isForceClose) => {
       //retrieve the output of the compilation
       const data = stats.compilation.assets['index.js'].source();
 
+      const secretKey = new Uint8Array(
+        signingKey.singingPrivateKey
+          .toString()
+          .match(/.{1,2}/g)
+          .map(byte => parseInt(byte, 16)),
+      );
+
       // update content data
       const dAppMetaData = getDappMetaData();
-      const content = JSON.stringify({
+      const content = {
         ...dAppMetaData,
         used_signing_key: signingKey.public_key,
         code: data,
-      });
+      };
+      let buff = Buffer.allocUnsafe(0);
+
+      const { name = {} } = content;
```

Comment:
all the logic that hash the bundle should be in the utils fail for better testing and single responsibility.

Reply:
done